boards/xtensa/esp32/esp32-wrover-kit/src/esp32_bringup.c

```diff
@@ -259,6 +264,24 @@ int esp32_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_SENSORS_BMP180
+  ret = board_bmp180_initialize(0, 0);
+
+  if (ret < 0)
+    {
+      syslog(LOG_ERR, "Failed to initialize BMP180
+                       Driver for I2C0: %d\n", ret);
+      ret2 = board_bmp180_initialize(0, 1);
```

Comment:
Why are you using two I2C buses to connect two devices?  And why 2 of the same device?

Reply:
@Ouss4 , I'm searching for BMP180 in both I2C buses because of possibility to enable 2 I2Cs in ESP32. When both i2C buses are enabled and BMP180 device driver is also enabled, with recourses we have today in menuconfig, there isn't a way to know in which I2C is the BMP180 wired in. This is why I search in both buses. Is it clear?

Reply:
I'm trying to cover a use-case where both I2Cs are used, but only one BMP180 will be there.

Reply:
> there isn't a way to know in which I2C is the BMP180 wired in

In boards we should know to what I2C bus a device is connected.  In some it's soldered.  Here, it's just the fact that you are using an external device that makes it possible to have it in two different buses.
What we do with external devices is to have instructions in the board's README.txt to explain how the connections were made and where the code expects the external device to be.

Reply:
Ok. In this case, does it mean I should expect BMP180 could be wired to just one I2C (for example, I2C0)?

Reply:
Yes, either I2Cs is okay.